### ISSUE TITLE
Add MOB-suite as a pipeline to IRIDA

### ISSUE THREAD

**Issue by apetkau** [phac-nml/irida-pipeline-plugins#4]
A request to add MOB-suite (https://github.com/phac-nml/mob-suite).

Related to https://github.com/phac-nml/mob-suite/issues/26

**Comment by kbessonov1984** [comment]
Thank you Aaron. Will start development of IRIDA plugin in a couple of weeks after the ecTyper patching

**Comment by apetkau** [comment]
Awesome. Sounds great :+1:

**Comment by apetkau** [comment]
Looks like @dfornika has been working on a MOB-suite plugin for IRIDA - https://github.com/dfornika/irida-plugin-mob-suite :smile:

**Comment by kbessonov1984** [comment]
Yes, we've talked about this with Dan a couple of weeks ago, but I was not aware that so much progress was already made as I was caught up with other projects (host-range bug polishing, ecTyper). We still need to upgrade from 1.4.9.1 version to 2.0 to include host range features in the new realease and, perhaps, provide additional options such as choice of an assembler (e.g. shovill). Big thanks to Dan for his work for saving us time! Will expand on his developments and notify each other.
